src/hypster/interactive/interact.css

```
@@ -0,0 +1,196 @@
+.hypster-widget {
+  --hypster-border: color-mix(in srgb, currentColor 14%, transparent);
+  --hypster-muted: color-mix(in srgb, currentColor 58%, transparent);
+  --hypster-surface: color-mix(in srgb, Canvas 96%, currentColor 4%);
+  --hypster-surface-strong: color-mix(in srgb, Canvas 90%, currentColor 10%);
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)